rasa/utils/tensorflow/environment.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import os
+from typing import Text, Tuple, Dict
+import warnings
+from rasa.constants import (
+    ENV_GPU_CONFIG,
+    ENV_CPU_INTER_OP_CONFIG,
+    ENV_CPU_INTRA_OP_CONFIG,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def setup_gpu_environment() -> None:
+    """Set configuration for a GPU environment based on the environment variable set"""
+
+    from tensorflow import config as tf_config
```

Comment:
@dakshvar22 Sorry, I thought about this one more time and we should only import this in case the env variable was actually set.

Reply:
Changed in this